betterlockscreen

```diff
@@ -154,7 +155,12 @@ lock() {
         --time-color="$timecolor" \
         --time-font="$font" \
         --time-size="$fontlg" \
-        --date-str "" \
+        --date-pos="ix-265:iy-38" \
```

Comment:
Yeah, let's just put that anywhere. No one will mind that it's gigantic or half hanging out of the box.